test/performance/statsd/parameters.yml

```diff
@@ -6,8 +6,8 @@ data_type: "metrics"
 
 # Number of metrics to be sent or number of log lines being written  each minute
 values_per_minute: "<values_per_minute>"
-# Number of seconds the agent should run and collect the metrics. In this case, 10 minutes
-agent_collection_period: 600 
+# Number of seconds the agent should run and collect the metrics. In this case, 5 minutes
+agent_collection_period: 300 
```

Comment:
Reduce to 5 minutes since the stress time for CWA has more than 3 minutes, enough to fill the buffer of CWA and increase more retry for validator since the timeout for running would be only 30 minutes. Therefore, allowing 3 retries at most. If you guys agree on reduce more, it would be good too !

Reply:
> Is this the time that the agent should run for the performance test?

Yes that is correct. The validator will wait [for 5 minutes before validating ](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/validator/validators/validator.go#L46)

Reply:
> I'm not understanding the parts about the validator retries. Are the number of retries being changed?

We are using Github Action to trigger the stress and the timeout minutes is 30. So if the test. run more than 30 minutes, the Github Action will make the test timeout. But the [validators has at least 5 retries](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/validator/main.go#L53-L63) and each time its run with 15 - 17 minutes for performance/stress testing. Therefore, we are able to run 2 retries; so decrease the runtime to allow more retries ( 2 -> 4)